core/shared/src/test/scala/cats/parse/ParserTest.scala

```diff
@@ -610,7 +610,7 @@ class ParserTest extends munit.ScalaCheckSuite {
 
   import ParserGen.{arbParser0, arbParser, biasSmall, genString, arbString}
 
-  val tests: Int = if (BitSetUtil.isScalaJs) 50 else 2000
+  val tests: Int = if (BitSetUtil.isScalaJvm) 2000 else 50
```

Comment:
Can we keep this condition `isScalaJs`? Is Scala Native runtime performance so slow?

Reply:
Actually, when I tested it before, I felt scala native is slower than JVM. I will check it again and if it is not so slower, I will put those conditions back.

Reply:
No problem. Well that's disappointing 😂

Reply:
Yeah, perhaps there are some performance bottle necks in Scala Native runtime.

Reply:
Ok, thanks for trying! Let's just leave it.

Anyway, the Native tests are not running with full optimizations enabled, perhaps that will be better.

Reply:
Ah, that may be the case. I don't think we should run every ci tests with full optimizations, which take quite long.

Reply:
And it turned out that it takes long to run tests on Scala Native if we set tests 2000 :(

Reply:
I hope Scala Native gets faster in future release🙁